app/frontend/wallet/shelley-wallet.ts

```diff
@@ -191,6 +191,30 @@ const ShelleyBlockchainExplorer = (config) => {
     return response
   }
 
+  async function getBestBlock() {
+    let bestBlock: number = 0
+    try {
+      const response = await request(
+        `${ADALITE_CONFIG.ADALITE_BLOCKCHAIN_EXPLORER_URL}/api/v2/bestBlock`
+      )
+      bestBlock = response.Right.bestBlock
+    } catch (e) {}
+
+    return bestBlock
+  }
+
+  async function getBestSlot() {
+    let bestSlot: number = 0
+    try {
+      const response = await request(
+        `${ADALITE_CONFIG.ADALITE_BLOCKCHAIN_EXPLORER_URL}/api/v2/bestSlot`
+      )
+      bestSlot = response.Right.bestSlot
+    } catch (e) {}
```

Comment:
Since you are not doing anything else in the try block, I would use `.catch(e)` syntax. See the implementation of the function above.

Reply:
Fixed